### PR TITLE
build: add missing attributes to Win installer

### LINKED ISSUE #17170: Windows installer properties and signing
Downloaded [`bitcoin-0.19.0rc1-win64-setup.exe`](https://bitcoincore.org/bin/bitcoin-core-0.19.0/test.rc1/bitcoin-0.19.0rc1-win64-setup.exe).

The Properties - Details tab lacks "File description" and "Copyright" values:

Screenshot from 2019-10-17 09-32-02

Also, the note that this `exe` file is signed by _Bitcoin Core Code Signing Association_ could be mentioned somewhere in our docs.

### PR DESCRIPTION
**Pull request by fanquake**
Fixes: #17170.

### CODE REVIEW

File: share/setup.nsi.in [hunk 1]
```diff
@@ -58,8 +58,8 @@ VIAddVersionKey ProductVersion "@PACKAGE_VERSION@"
 VIAddVersionKey CompanyName "${COMPANY}"
 VIAddVersionKey CompanyWebsite "${URL}"
 VIAddVersionKey FileVersion "@PACKAGE_VERSION@"
-VIAddVersionKey FileDescription ""
-VIAddVersionKey LegalCopyright ""
+VIAddVersionKey FileDescription "Installer for @PACKAGE_NAME@"
+VIAddVersionKey LegalCopyright "@COPYRIGHT_HOLDERS_FINAL@"
```
Comment (hebasto) [inline share/setup.nsi.in]: Could be
`Copyright (C) 2009-@COPYRIGHT_YEAR@ @COPYRIGHT_HOLDERS_FINAL@`
?

**Review by hebasto — COMMENTED** [review]
Concept ACK.

File: share/setup.nsi.in (hunk 1 above)
Comment (fanquake) [reply share/setup.nsi.in]: I don't really care, but sure.

**Comment by luke-jr** [comment]
utACK

**Comment by hebasto** [comment]
ACK 6c223152238d2e818e38357b03f38a4dbe9de016, I have reviewed the code and it looks OK, I agree it can be merged.

**Pull request merged**